File: GNU Octave/Edge of the Ocean/makeArrayConsecutive2.m
function res = makeArrayConsecutive2 (statues)

    MIN = min(statues);
    MAX = max(statues);
    
    %% +2 => MIN & MAX don't count
    %% -1 => There are MAX - MIN - 1 numbers between any two integer values
    
    res = MAX - MIN - numel(statues) + 2 - 1

endfunction

File: JavaScript/Edge of the Ocean/makeArrayConsecutive2.js
function makeArrayConsecutive2(statues) {
    let max = -1, min = 21;
    statues.forEach((statue) => 
    {
        if(statue > max)
            max = statue;
        if(statue < min)
            min = statue;
    })
    let sequence = max - min + 1;
    return sequence - statues.length;
}